### PR TITLE
update to add rvvi.sleep

### PR DESCRIPTION
**Pull request by eroom1966**
@MikeOpenHWGroup @silabs-mateilga 
Hi Mike, Marton

As promised I have a PR which includes a sleep signal in the RVVI interface, lets see if this compared with the one generated in the RVFI when implemented

### CODE REVIEW

File: vendor_lib/imperas/imperas_DV_COREV/ChangeLog.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CV32E40P Change Log
```
Comment (MikeOpenHWGroup) [inline vendor_lib/imperas/imperas_DV_COREV/ChangeLog.md:1]: This changelog mentions the E40P, but I'm guessing it applies to all E40 cores.   You can fix that in a future PR if you like.

**Comment by MikeOpenHWGroup** [comment]
AFAICT, this is not intended to fully resolve #1193, but i did a local merge and ran both a ci_check (passed) and 100 runs of `corev_rand_interrupt_wfi_mem_stress`.  There were three failures, all with the same signature: after a number of instructions (43,545, 162,359 and 7,454) the tests hang and eventually is terminated by the Global timeout.   I have not investigated to see what caused the hang.  There were no miscompares in any of the tests.

**Comment by eroom1966** [comment]
@MikeOpenHWGroup I have also added the deferint fix into the haltreq code that drives the signals into the REF model

**Review by silabs-mateilga — APPROVED** [review]
The changes looks good, and no new or unknown issues on a full regression run with xrun.

**Pull request merged**